.ci/jenkins/dsl/test.sh

```diff
@@ -1,9 +1,29 @@
 #!/bin/bash -e
 
-script_dir_path=$(cd `dirname "${BASH_SOURCE[0]}"`; pwd -P)
+# Used to retrieve current git author, to set correctly the main config file repo
+GIT_SERVER='github.com'
 
-export DSL_DEFAULT_MAIN_CONFIG_FILE_REPO=kiegroup/optaplanner
-export DSL_DEFAULT_MAIN_CONFIG_FILE_REF=main
+git_url=$(git remote -v | grep origin | awk -F' ' '{print $2}' | head -n 1)
+if [ -z "${git_url}" ]; then
+  echo "Script must be executed in a Git repository for this script to run currently"
```

Comment:
Does the 'currently' mean it's going to change? Or should it have been 'correctly'?

Reply:
oh I think this is a mistake... it should be `correctly`

Reply:
```suggestion
  echo "Script must be executed in a Git repository for this script to run correctly"
```